### ISSUE TITLE
Add support for Pixel Portrait Mode

### ISSUE THREAD

**Issue by tibbi** [SimpleMobileTools/Simple-Gallery#490]

**Comment by ocram** [comment]
Has this been closed because the feature has already been implemented, or because you found it not worthwhile to add it?

**Comment by tibbi** [comment]
looks like by mistake, as there is no other report about it. Anyway, it hasnt been implemented yet, but I want to do it. It just isnt easy at all.

**Comment by ocram** [comment]
Great, thanks!

Considering only the Pixels’ market share, you could simply ignore these devices. But no Pixel owner will ever use your fantastic gallery app if “extensions” such as this one here or [other ones](https://github.com/SimpleMobileTools/Simple-Gallery/issues/1426) are not supported, because it just breaks basic device features.

As for the portrait mode feature, this is implemented by replacing a single photo with a folder of the same name (minus the extension) that then contains both the original version and the “enhanced” version, right?

**Comment by tibbi** [comment]
yeah, it creates new folders for every image. But it isnt just a Pixel feature, I have it on OnePlus too. Probably most newer devices will have it, so I definitely dont want to ignore it.
